### ISSUE TITLE
[Feature Request] el-popover to show after content is entirely fetched

### ISSUE THREAD

**Issue by VinceBassman** [ElemeFE/element#9856]
### Existing Component
Yes

### Component Name
el-popover

### Description
If the content of the popover has to be fetched e.g. from one API popover show first an empty tooltip and load the content as soon as it has been fetched.
The position of the popover is not readjusted after the content has been loaded but remains based on the empty content from the beginning.

Repro link:
https://jsfiddle.net/dLwcsdj1/

And how it is expected to be after the content has loaded (from the position):
https://jsfiddle.net/5tjcxj69/

**Comment by Leopoldthecoder** [comment]
You can call `updatePopper` after data is fetched: https://jsfiddle.net/leopoldthecuber/dLwcsdj1/5/ . `updatePopper` is an inner method, but is stable enough to be used in production.

**Comment by VinceBassman** [comment]
That's it! Thanks for the quick answer!

**Comment by bq-hentai** [comment]
I have met something similar today. And I debugged, then found 
![image](https://user-images.githubusercontent.com/19964297/36960943-20da1d2c-2084-11e8-9e4f-5d9de9b9a52c.png).

If `query` changes, the `updatePopper` is called. Just assume `nextTick` is called via microTask, and if we send a xhr, and won't update at last. And the first update is useless maybe?

And I tried to call `updatePopper` manually and fixed.

I just found the issue and post my experience. But I don't think it's an elegant way. Can we have another better method to do this or enhance the api?

Anyway, thx for ur great work on Element-ui.
